src/components/tabs/Tabsv2.stories.tsx

```diff
@@ -0,0 +1,108 @@
+                    <TabPane id={"cloud"}>{"CLOUD"}</TabPane>
+                    <TabPane id={"infra"}>{"INFRASTRUCTURE"}</TabPane>
+                    <TabPane id={"metrics"}>{"METRICS"}</TabPane>
+                    <TabPane id={"policies"}>{"POLICIES"}</TabPane>
+                </TabsV2>
+            )}
+        </State>
+    ))
+    .add("TabV2 Overflow", () => (
+        <State store={store}>
+            {(state) => (
+                <TabsV2
+                    id="overflowTabs"
+                    tabs={state.simpleTabs}
+                    tabOrientation={TabV2Orientation.HORIZONTAL}
+                    overflowTabsFrom={2}
+                >
+                    <TabPane id={"dashboard"}>{"DASHBOARD"}</TabPane>
+                    <TabPane id={"mgmt"}>{"MANAGEMENT"}</TabPane>
+                    <TabPane id={"cloud"}>{"CLOUD"}</TabPane>
+                    <TabPane id={"infra"}>{"INFRASTRUCTURE"}</TabPane>
+                    <TabPane id={"metrics"}>{"METRICS"}</TabPane>
+                    <TabPane id={"policies"}>{"POLICIES"}</TabPane>
+                </TabsV2>
+            )}
+        </State>
+    ))
+    .add("TabV2 Overflow - only tabs", () => (
+        <State store={store}>
+            {(state) => (
+                <TabsV2
+                    id="overflowTabs"
+                    tabs={state.simpleTabs}
+                    tabOrientation={TabV2Orientation.HORIZONTAL}
+                    overflowTabsFrom={2}
+                />
+            )}
+        </State>
+    ))
+    .add("TabV2 Static with Tab Selected", () => (
+        <State store={store}>
+            {(state) => (
+                <TabsV2
+                    id="staticTabsWithTabSelected"
+                    tabs={state.staticTabsWithTabSelected}
+                    tabOrientation={TabV2Orientation.HORIZONTAL}
+                >
+                    <TabPane id={"dashboard"}>{"DASHBOARD"}</TabPane>
+                    <TabPane id={"mgmt"}>{"MANAGEMENT"}</TabPane>
+                    <TabPane id={"cloud"}>{"CLOUD"}</TabPane>
+                    <TabPane id={"infra"}>{"INFRASTRUCTURE"}</TabPane>
+                    <TabPane id={"metrics"}>{"METRICS"}</TabPane>
+                    <TabPane id={"policies"}>{"POLICIES"}</TabPane>
+                </TabsV2>
+            )}
+        </State>
+    ))
+    .add("TabV2 Vertical", () => (
+        <State store={store}>
+            {(state) => (
+                <TabsV2 id="verticalTabs" tabs={state.simpleTabs} tabOrientation={TabV2Orientation.VERTICAL}>
+                    <TabPane id={"dashboard"}>{"DASHBOARD"}</TabPane>
+                    <TabPane id={"mgmt"}>{"MANAGEMENT"}</TabPane>
+                    <TabPane id={"cloud"}>{"CLOUD"}</TabPane>
+                    <TabPane id={"infra"}>{"INFRASTRUCTURE"}</TabPane>
+                    <TabPane id={"metrics"}>{"METRICS"}</TabPane>
+                    <TabPane id={"policies"}>{"POLICIES"}</TabPane>
+                </TabsV2>
```

Comment:
can we create common function to return these TabPanes ?

Reply:
Done